### PR TITLE
Ai 15 setup django app

### PR DESCRIPTION
**Pull request by SlawCzech**
# Description

Init django template with frontend for Azeno Interview. 

# How Has This Been Tested?

Manual run and testing, pytest.

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my code
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] New and existing unit tests pass locally with my changes
- [X] The related memme has been added

Fixed it

### CODE REVIEW

File: interview/backend/config/settings.py [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+    "ACCESS_TOKEN_LIFETIME": timedelta(minutes=5),
+    "REFRESH_TOKEN_LIFETIME": timedelta(days=25),
+    "ROTATE_REFRESH_TOKENS": True,
+    "BLACKLIST_AFTER_ROTATION": True,
+    # TODO: If updating last login in the table is desired, throttling must be set up in django rest framework
+    "UPDATE_LAST_LOGIN": False,
+    "ALGORITHM": "HS256",
+    "SIGNING_KEY": os.environ.get("JWT_SECRET_KEY", SECRET_KEY),
+    "VERIFYING_KEY": "",
+    "AUDIENCE": "azeno_quiz",
```
Comment (MWasile) [inline interview/backend/config/settings.py:185]: should be renamed to azeno interview

File: interview/web/index.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Azeno Quiz</title>
```
Comment (MWasile) [inline interview/web/index.html:7]: and this

File: interview/web/package.json [hunk 3]
```diff
@@ -0,0 +1,66 @@
+{
+    "name": "azeno-quiz",
```
Comment (MWasile) [inline interview/web/package.json:2]: and this

**Review by MWasile — COMMENTED** [review]
a few minor changes and it will be fine 😃

File: interview/backend/config/settings.py (hunk 1 above)
Comment (SlawCzech) [reply interview/backend/config/settings.py]: done

File: interview/web/index.html (hunk 2 above)
Comment (SlawCzech) [reply interview/web/index.html]: done

File: interview/web/package.json (hunk 3 above)
Comment (SlawCzech) [reply interview/web/package.json]: done

File: interview/backend/.dockerignore [hunk 4]
```diff
@@ -0,0 +1,5 @@
+.idea
+.vscode
+*/**.pyc
```
Comment (BartekSzymik) [inline interview/backend/.dockerignore:3]: jeszcze to Sławku popraw na szybko **/*.pyc i będzie super

**Review by BartekSzymik — COMMENTED** [review]
Good job!

**Pull request merged**